### PR TITLE
Add ami environment file

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

File: ami_environment.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - jupyter
+  - matplotlib
+  - scikit-learn>=0.19
+  - mahotas
+  - scikit-image
+  - keras>=2.0.0
+  - h5py
+  - tensorflow
+  - pytorch-cpu
+  - torchvision-cpu
```
Comment (jorisvandenbossche) [inline ami_environment.yml]: Those pytorch-cpu and torchvision-cpu was copied from the ami_environment.yml file of the autism challenge. But I suppose here we want the gpu version? (I don't know what the default pytorch installs with conda?)

File: ami_environment.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: root
+channels:
+  - pytorch
```
Comment (jorisvandenbossche) [inline ami_environment.yml:3]: Any idea if this is needed? pytorch is also available from the default channel, but not sure the difference 

(the installation instructions of pytorch seem to be to install from this channel, so maybe therefore keep it)

**Review by jorisvandenbossche — COMMENTED** [review]
cc @mehdidc @aboucaud @glemaitre

**Comment by glemaitre** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  you need the pytorch channel because you have both version. This is provided by pytorch. For the gpu version I would assume to switch cpu to gpu but it is documented on the Web of pytorch.                                                                                                                                                                         Sent from my phone - sorry to be brief and potential misspell.

File: ami_environment.yml (hunk 2 above)
Comment (aboucaud) [reply ami_environment.yml:3]: can you add `cuda90` as a requirement ? otherwise it downgrades to the 8.0 version

File: ami_environment.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - mahotas
+  - scikit-image
+  - keras>=2.0.0
+  - h5py
+  - tensorflow-gpu
+  - pytorch
+  - torchvision
+  - pip:
+      - https://api.github.com/repos/paris-saclay-cds/ramp-workflow/zipball/master
+      - git+https://github.com/paris-saclay-cds/ssd_keras
```
Comment (aboucaud) [inline ami_environment.yml]: maybe this last one can also be installed via the zipball for consistency, no ?

**Comment by aboucaud** [comment]
Alright, I need this to create the AMI.

**Comment by aboucaud** [comment]
Looking good, thanks.

**Pull request merged**